worker/oci/resolvconf.go

```diff
@@ -0,0 +1,81 @@
+package oci
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/docker/libnetwork/resolvconf"
+	"github.com/moby/buildkit/util/flightcontrol"
+	"golang.org/x/net/context"
+)
+
+var g flightcontrol.Group
+var notFirstRun bool
+var lastNotEmpty bool
+
```

Comment:
Don't we need a mutex?

Reply:
flightcontrol callback gets called in a single thread